packages/std/src/lib.rs

```diff
@@ -1,6 +1,3 @@
-#![cfg_attr(feature = "backtraces", feature(error_generic_member_access))]
-#![cfg_attr(feature = "backtraces", feature(provide_any))]
-
```

Comment:
We probably want to export BT here to allow people to create the StdError cases manually. On the other hand, we have the constructors for that. Thoughts?

Reply:
I guess there is no harm in exporting it, but the only real use-cases I can see is having backtraces in contract errors. For creating the errors, the constructors seem good enough to me. The only reason to create one manually would be to capture a BT to put it into an error later, which doesn't make sense to me.

Reply:
okay, then let's not export it for now and see if we have a good reason to change that later